core/drivers/crypto/hisilicon/include/sec_main.h

```diff
@@ -0,0 +1,376 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/* Copyright (c) 2022-2024 HiSilicon Limited. */
+#ifndef __SEC_MAIN_H
+#define __SEC_MAIN_H
+
+#include "hisi_qm.h"
+
+#include <initcall.h>
```

Comment:
Please include _initcall.h_ from _core/drivers/crypto/hisilicon/sec_main.c_
Instead, explicitly include _stdint.h_ and _compiler.h_